keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -180,65 +187,51 @@ be read as-is, i.e. there will not be any implicit isolation effects for the Pod
 selected by the AdminNetworkPolicy, as opposed to what NetworkPolicy rules imply.
 
 - Pass: Traffic that matches a `Pass` rule will skip all further rules from all
-  positive priority (non-zero) numbered ANPs and instead be enforced by the K8s NetworkPolicies.
-  If there is no K8s NetworkPolicy rule match, and no ANP priority "0" rule
-  match (more on this in the [priority section](#priority)), traffic will be governed
-  by the implementation. For most implementations, this means "allow", but there
-  may be implementations which have their own policies outside of the standard
-  Kubernetes APIs.
+  lower precedenced ANPs, and instead be enforced by the K8s NetworkPolicies.
+  If there is no K8s NetworkPolicy rule match, and no BaselineAdminNetworkPolicy
+  rule match (more on this in the [priority section](#priority)), traffic will be
+  governed by the implementation. For most implementations, this means "allow",
+  but there may be implementations which have their own policies outside of the
+  standard Kubernetes APIs.
```

Comment:
sometimes it's nice to not rewrap paragraphs after making small edits to one line, to avoid irrelevant diffs...